projects/pastanaga/src/lib/controls/filtered-checkbox-group/filtered-checkbox-group.component.scss

```diff
@@ -0,0 +1,86 @@
+@import "../../styles/variables";
+
+:host {
+    position: relative;
+    display: block;
+
+    legend {
+        pa-badge {
+            display: inline-block;
+            position: relative;
+            top: rythm(.25);
+            left: rythm(1.5);
+        }
+        & > span {
+            display: inline-block;
+            margin-top: rythm(.25) * 3;
+        }
+    }
+
+    .filter-container {
+        padding-top: rythm(.5);
+        pa-icon {
+            background-color: rgba($blue, .06);
+            display: block;
+            height: rythm(4);
+            line-height: rythm(4);
+            float: left;
+            border-top-left-radius: rythm(.25);
+            border-bottom-left-radius: rythm(.25);
+            ::ng-deep svg {
+                height: rythm(2);
+                width: rythm(2);
```

Comment:
check if this size is not one of default sizes for icons. I wouldn't be surprised if you could get it from pa-icon attributes/configuration

Reply:
yes that's medium but it cannot be set directly (only via the `icon` input)
we will have to improve that but not in this PR